samples/suit/smp_transfer/sysbuild/nrf54h20dk_nrf54h20_memory_map.dtsi

```diff
@@ -5,13 +5,13 @@
  */
 
 &cpuapp_rx_partitions {
-	/delete-node/ partition@a6000;
-	cpuapp_slot0_partition: partition@a5000 {
-		reg = <0xa5000 DT_SIZE_K(256)>;
+	/delete-node/ partition@94000;
+	cpuapp_slot0_partition: partition@94000 {
```

Comment:
Shouldn't that be `0x93000`?

Reply:
As far as I can see it is correct.

cpurad [0x54000 - 0x7c000>  (size 0x28000)
radrec [0x7c000 - 0x94000>  (size 0x17000)
cpuapp [0x94000 - 0xd4000>  (size 0x40000)

so both cpurad and cpuapp start addresses match the "normal" layout (snippet from hello world)

```
			cpurad_rx_partitions: cpurad-rx-partitions {
				cpurad_slot0_partition: partition@54000 {
					reg = < 0x54000 0x40000 >;
				};
			};
			cpuapp_rx_partitions: cpuapp-rx-partitions {
				cpuapp_slot0_partition: partition@94000 {
					reg = < 0x94000 0x50000 >;
				};

```

Reply:
Well do your math, this isn't correct 

radrec [0x7c000 - 0x94000> (size 0x17000)
cpuapp [0x94000 - 0xd4000> (size 0x40000)

since 0x7c000+0x17000 = 0x93000, so you are right

Reply:
should be fixed now